src/lvl_script_commands.c

```diff
@@ -35,6 +35,9 @@
 #include "creature_states_mood.h"
 #include "room_util.h"
 #include "creature_instances.h"
+#include "lvl_script_value.h"
```

Comment:
since both new lines would be better in thing_list.h this include might not be needed then, so check if it does